main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+class Fruit{
+private:
+    string name;
+    string size;
+    string taste;
+    string color;
+public:
+    
+
+    
+    Fruit(string name,string size,string taste,string color){
+        this->name = name;
+        this->size = size;
+        this->color = color;
+        this->taste = taste;
+    }
+    
+    friend ostream& operator<< (ostream &out, const Fruit &f){
```

Comment:
const "Fruit" should be written in lower case, please, change the letter to lowercase😊❤️

Reply:
fixed